content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -529,11 +529,15 @@ world.register_system(|world: &mut World| { /* do anything */ });
 
 All these improvements round out one-shot systems significantly and they should now behave normally in any Bevy context.
 
-## WGPU Upgrade
+## WGPU Upgrade and Renderer Improvements
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @Elabajaba</div>
 
-TODO.
+In Bevy 0.13 we upgraded from wgpu 0.17 to wgpu 0.19, which includes the long awaited wgpu [arcanization](https://gfx-rs.github.io/2023/11/24/arcanization.html) that allows us to do things like compiling shaders asynchronously to avoid shader compilation stutters, and to multithread draw call creation for better performance.
```

Comment:
```suggestion
In Bevy 0.13 we upgraded from `wgpu` 0.17 to `wgpu` 0.19, which includes the long awaited `wgpu` [arcanization](https://gfx-rs.github.io/2023/11/24/arcanization.html) that allows us to do things like compiling shaders asynchronously to avoid shader compilation stutters, and to multithread draw call creation for better performance.
```